doc/redesign/README.md

```diff
@@ -0,0 +1,33 @@
+# Redesign Documentation
+
+## Summary
+
+The first step to redesign Jangouts was to understand its users. Thus, we conducted an online [Survey][1] and created a few [Personas][2], profiles of fictional  users to represent different user types who use Jangouts. Having analyzed the [Results][3] of the survey, we matched the respondents to the personas to recognize the needs of specific types of users that are described in [Personas' Requirements][4]. Then, we specified the requirements for the new design in [UI Requirements Specification][5] and began designing wireframes. The [First Draft][6] shows 10 different ideas that were later narrowed down and explained in the [Wireframes Design Explanation][7] that lead to the [Second Draft][7]. This gave a foundation for the design of the Prototype that can be statically viewed [here][9] or interactively tested [here][10].
+
+## Table of contents
+
+1. Gathering Requirements
+   - [Survey][1]
+   - [Personas][2]
+2. Describing Requirements
+   - [Survey Results][3]
+   - [Personas' Requirements][4]
+   - [UI Requirements Specification][5]
+3. Designing Wireframes
+   - [First Draft][6]
+   - [Wireframes Design Explanation][7]
+   - [Second Draft][8]
+5. Prototype
+   - [Static][9]
+   - [Interactive][10]
+
+[1]: ./survey.md
+[2]: ./personas.md
+[3]: ./survery-results.md
+[4]: ./personas-requirements.md
+[5]: ./ui-requirements-spec.md
+[6]: ./wireframes/draft1
+[7]: ./wireframes-design-explanations.md
+[8]: ./wireframes/draft2
+[9]: ./prototype/jangout-prototype.pdf
```

Comment:
`jangout-prototype.pdf` should be `jangouts-prototype.pdf` (note the missing 's').